applications/RANSApplication/custom_elements/data_containers/k_epsilon/k_element_data.h

```diff
@@ -67,33 +83,29 @@ class KElementData : public ConvectionDiffusionReactionElementData
         const Matrix& rShapeFunctionDerivatives,
         const int Step = 0);
 
-    array_1d<double, 3> CalculateEffectiveVelocity(
-        const Vector& rShapeFunctions,
-        const Matrix& rShapeFunctionDerivatives) const;
+    ///@}
 
-    double CalculateEffectiveKinematicViscosity(
-        const Vector& rShapeFunctions,
-        const Matrix& rShapeFunctionDerivatives) const;
-
-    double CalculateReactionTerm(
-        const Vector& rShapeFunctions,
-        const Matrix& rShapeFunctionDerivatives) const;
+protected:
+    ///@name Protected Members
+    ///@{
 
-    double CalculateSourceTerm(
-        const Vector& rShapeFunctions,
-        const Matrix& rShapeFunctionDerivatives) const;
+    using BaseType::mEffectiveVelocity;
+    using BaseType::mEffectiveKinematicViscosity;
+    using BaseType::mReactionTerm;
+    using BaseType::mSourceTerm;
```

Comment:
unnecessary

Reply:
As I explained, I think it is necessary if you want to avoid calling `this->` to get base class protected variables. Is there any other way?